code/game/gamemodes/objective.dm

```diff
@@ -1019,3 +1025,4 @@ GLOBAL_LIST_EMPTY(possible_items)
 	var/area/target_area = get_area(target)
 
 	return (istype(user_area, dropoff) && istype(target_area, dropoff))
+
```

Comment:
Don't add empty new lines at the end of files that don't need them, when it's /tg/ files.
```suggestion
```